cmake/FindTime.cmake

```diff
@@ -0,0 +1,13 @@
+include(FindPackageHandleStandardArgs)
+
+find_program(Time_EXECUTABLE NAMES time)
+
+find_package_handle_standard_args(Time REQUIRED_VARS Time_EXECUTABLE)
+
+if (Time_FOUND)
+  mark_as_advanced(Time_EXECUTABLE)
+  add_executable(Time IMPORTED)
+  set_property(TARGET Time PROPERTY IMPORTED_LOCATION ${Time_EXECUTABLE})
+else()
+  message(WARNING "ctest will skip tests test_xdd_{passdelay/startdelay}. Intall time command through package manager")
+endif()
```

Comment:
This file is probably not necessary. `find_program` should be enough.

Reply:
Yeah, that is what I initially was doing. I will reevaluate this again to see if I can trim off all the fat.

Reply:
I updated this. I just use `find_program()` now in the CMakeLists.txt in the test directory.